.github/workflows/main.yml

```diff
@@ -204,9 +204,17 @@ jobs:
 
       - name: Archive ARM artifacts
         run: |
-          tar -czf artifacts-arm64.tar.gz -C artifacts-arm64 .
-          echo "Created artifacts-arm64.tar.gz"
-          ls -lh artifacts-arm64.tar.gz
+          tar -czf midnight-node-arm64.tar.gz -C artifacts-arm64 .
+          echo "Created midnight-node-arm64.tar.gz"
+          ls -lh midnight-node-arm64.tar.gz
+
+      - name: Download release notes artifact
+        uses: actions/download-artifact@v4
+        with:
+          name: release-notes
+          path: .
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          run-id: ${{ github.event.workflow_run.id || github.run_id }}
```

Comment:
Land https://github.com/midnightntwrk/midnight-node/pull/384 first and then you don't need this.